src/utils/localizedTypes.ts

```diff
@@ -93,11 +113,9 @@ const toMenuEntry = (
     referenceStrings.push(parentCommonName)
   }
 
-  const cultivarIndex = scientificName?.indexOf("'")
-  if (cultivarIndex !== -1) {
-    const cultivarName = scientificName
-      .substring(cultivarIndex)
-      .replaceAll("'", '')
+  const cultivar = extractCultivar(scientificName)
+  if (cultivar) {
+    const cultivarName = cultivar.replaceAll("'", '')
```

Comment:
This is wrong, because cultivars can contain `'` internally (e.g. `Malus pumila 'Ashmead's Kernel'`). And I would move the spirit of this logic into `extractCultivar` so that that function returns a clean cultivar name.